### PR TITLE
Update changelog for 0.53.1

### PR DESCRIPTION
**Pull request by esc**
As title

### CODE REVIEW

File: CHANGE_LOG [hunk 1]
```diff
@@ -3,6 +3,26 @@ Version 0.54.0
...
+
+This is a bugfix release for 0.53.0. It contains the following four
+pull-requests which fix two critical regressions and two build failures
+reported by the openSuSe team:
+
+* PR #6851 set non-reported llvm timing values to 0.0
+* PR #6837 Ignore warnings from packaging module when testing import behaviour.
+* PR #6828 Fix regression in CUDA: Set stream in mapped and managed array
+  device_setup
+* PR #6826 Fix regression on gufunc serialization
```
Comment (stuartarchibald) [inline CHANGE_LOG:17]: Order is usually lowest to highest?

Comment (esc) [reply CHANGE_LOG:17]: ok, let me fix

Comment (esc) [reply CHANGE_LOG]: fixed

**Review by stuartarchibald — APPROVED** [review]
Thanks for the patch!

**Pull request merged**